src/private/user-page/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react';
+import { Col, Container, Row } from 'reactstrap';
+import Preloader from '../../components/preloader';
+import { getUserStart } from '../../actions/actions';
+import { useDispatch, useSelector } from 'react-redux';
+
+function UserPage () {
+  const dispatch = useDispatch();
+  const { user, preloader } = useSelector(state => state.user);
+
+  useEffect(() => {
+    dispatch(getUserStart());
```

Comment:
```suggestion
    dispatch(initialize());
    return () => clear()
```

META
CLEAR
INITIALIZE